Doc/library/sqlite3.rst

```diff
@@ -1236,17 +1303,24 @@ Connection objects
 
    .. attribute:: isolation_level
 
-      This attribute controls the :ref:`transaction handling
-      <sqlite3-controlling-transactions>` performed by :mod:`!sqlite3`.
+      Legacy control of the :ref:`transaction handling
+      <sqlite3-transaction-control-isolation-level>` performed by :mod:`!sqlite3`.
```

Comment:
```suggestion
      Controls the :ref:`legacy transaction handling mode
      <sqlite3-transaction-control-isolation-level>` of :mod:`!sqlite3`.
```

Re-reading this, I found the current revision of the syntax quite confusing and hard to understand and follow. I've tweaked it to be closer to the original and be much easier to read.

Reply:
Definitely easier to read. Thanks.

Reply:
Yeah, I haven't pushed it yet. I was waiting for the last unresolved conversation to resolve :)